From 4ab92ce748ffc9f408054360452064d4bf4e58ec Mon Sep 17 00:00:00 2001
From: Martin Kepplinger <martink@posteo.de>
Date: Tue, 27 Nov 2018 09:51:58 +0100
Subject: [PATCH 1/2] ts_config_filter: use format argument with sprintf

---
 src/ts_config_filter.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/ts_config_filter.c b/src/ts_config_filter.c
index 4eae1d8..d5b5c5a 100644
--- a/src/ts_config_filter.c
+++ b/src/ts_config_filter.c
@@ -89,13 +89,13 @@ struct ts_module_conf *ts_conf_get(struct tsdev *ts)
 		if (!conf_next->name)
 			goto fail;
 
-		sprintf(conf_next->name, modulebuf[i]);
+		sprintf(conf_next->name, "%s", modulebuf[i]);
 
 		conf_next->params = calloc(1, LINE_MAX);
 		if (!conf_next->params)
 			goto fail;
 
-		sprintf(conf_next->params, parambuf[i]);
+		sprintf(conf_next->params, "%s", parambuf[i]);
 
 		conf_next->raw = raw[i];
 
-- 
2.19.1

